### PR TITLE
Remove sentinel ping functionality

### PR DESCRIPTION
**Pull request by codablock**
This is not available in Dash Core 0.14.0

### CODE REVIEW

**Review by UdjinM6 — APPROVED** [review]
utACK

File: lib/config.py
Comment (UdjinM6) [inline lib/config.py]:
```
/lib/config.py:15:1: E302 expected 2 blank lines, found 1
def get_dash_conf():
^
The command "find ./lib ./test ./bin -name \*.py -exec pycodestyle --show-source --ignore=E501,E402,E722,E129,W503,W504 {} +" exited with 1.
```
https://travis-ci.org/dashpay/sentinel/jobs/511543379#L518

Comment (codablock) [reply lib/config.py]: :+1: Fixed this

**Review by UdjinM6 — APPROVED** [review]
re-utACK

**Comment by codablock** [comment]
Strange, after merging #63 this PR became unmergable even though there are not conflicts. When I click on "Update branch", it merges in develop...which I really don't like as I'd prefer a rebase. Did the rebase now and force-pushed, but this removed the ACKs...

@nmarley maybe there is some strange Github setting in-place?

**Comment by UdjinM6** [comment]
@codablock you have to wait for Travis build to succeed before you'll be able to merge.

**Pull request merged**
